### PR TITLE
[6.4] FIX UI provisioning template clone

### PR DESCRIPTION
**Pull request by ldjebran**
```
(robottelo) dlezz@dlezz:~/projects/robottelo-fork$ pytest -v tests/foreman/ui_airgun/test_provisioningtemplate.py::test_positive_clone 
============================================ test session starts =============================================
platform linux -- Python 3.6.6, pytest-3.6.1, py-1.5.4, pluggy-0.6.0 -- /home/dlezz/.pyenv/versions/3.6.6/envs/robottelo/bin/python
cachedir: .pytest_cache
shared_function enabled - ON - scope: ffff - storage: file
rootdir: /home/dlezz/projects/robottelo-fork, inifile:
plugins: wait-for-1.0.9, xdist-1.22.5, services-1.2.1, mock-1.10.0, forked-0.2, cov-2.5.1
collecting 1 item                                                                                            2018-09-19 13:14:07 - conftest - DEBUG - BZ deselect is disabled in settings

collected 1 item                                                                                             

tests/foreman/ui_airgun/test_provisioningtemplate.py::test_positive_clone PASSED                       [100%]

========================================= 1 passed in 53.54 seconds ==========================================
```

### CODE REVIEW

File: tests/foreman/ui_airgun/test_provisioningtemplate.py [hunk 1]
```diff
@@ -68,8 +68,5 @@ def test_positive_clone(session):
         assert session.provisioningtemplate.search(
             clone_name)[0]['Name'] == clone_name
         template = session.provisioningtemplate.read(clone_name)
-        cloned_template_os_list = [
-            el.split()[0] for el
```
Comment (abalakh) [inline tests/foreman/ui_airgun/test_provisioningtemplate.py]: What was the root cause of failure? Theoretically, if we use os name + split os title on ui and pick first element - they should be equal. Your approach is better anyways, i'm just curious

Comment (ldjebran) [reply tests/foreman/ui_airgun/test_provisioningtemplate.py]: in the UI the OS is listed as title not name, and we can use that directly without any other manipulations.

Comment (ldjebran) [reply tests/foreman/ui_airgun/test_provisioningtemplate.py]: seems this has been changed, since the test was written.

Comment (abalakh) [reply tests/foreman/ui_airgun/test_provisioningtemplate.py]: So the root cause wasn't here, but in `session.provisioningtemplate.clone` command - clone screen contained full os name with version but we were searching for name only. Change from os name to os title resolved the issue.

**Pull request merged**